src/test/java/test/IngressTest.java

```diff
@@ -210,4 +210,56 @@ void jira_ingress_path_default(Product product) throws Exception {
         }
     }
 
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "confluence")
+    void confluence_ingress_path_contextPath(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.create", "true",
+                "ingress.host", "myhost.mydomain",
+                "confluence.service.contextPath", "/confluence-tmp"));
+
+        final var ingresses = resources.getAll(Kind.Ingress);
+        Assertions.assertNotEquals(0, ingresses.size());
+
+        for (KubeResource ingress : ingresses) {
+            assertThat(ingress.getNode("spec", "rules").required(0).path("http").path("paths").required(0).path("path"))
+                    .hasTextContaining("/confluence-tmp");
+        }
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "confluence")
+    void confluence_ingress_path_value(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.create", "true",
+                "ingress.host", "myhost.mydomain",
+                "confluence.service.contextPath", "/context",
+                "ingress.path", "/ingress"));
+
+        final var ingresses = resources.getAll(Kind.Ingress);
+        Assertions.assertNotEquals(0, ingresses.size());
+
+        for (KubeResource ingress : ingresses) {
+            assertThat(ingress.getNode("spec", "rules").required(0).path("http").path("paths").required(0).path("path"))
+                    .hasTextContaining("/ingress");
+        }
+    }
+
+    @ParameterizedTest
+    @EnumSource(value = Product.class, names = "confluence")
+    void confluence_ingress_path_default(Product product) throws Exception {
+        final var resources = helm.captureKubeResourcesFromHelmChart(product, Map.of(
+                "ingress.create", "true",
+                "ingress.host", "myhost.mydomain"));
+
+        final var ingresses = resources.getAll(Kind.Ingress);
+        Assertions.assertNotEquals(0, ingresses.size());
+
+        for (KubeResource ingress : ingresses) {
+            assertThat(ingress.getNode("spec", "rules").required(0).path("http").path("paths").required(0).path("path"))
+                    .hasTextContaining("/");
```

Comment:
We should use `hasTextEqualTo` instead of `hasTextContaining`. For example: `"/foo/".hasTextContaining("/")` is going to pass, and this is not what we want.

Reply:
You are right, but the reason for using `hasTextContains` in this case and all other tests is we have more than one Node. One is `<path>` and another is `<path>/setup` (consider this assertion is in the loop)
I have changed the code slightly to use `equal` assertion and in the same time don't violate the test conditions in loop by dropping the `/setup` from the path if any is existed.